README.md

```diff
@@ -33,51 +33,36 @@ support AMD CPU, ARM CPU, and NVidia GPU through ONNX Runtime with limited testi
 
 ## Installation
 ### Install Framework
```

Comment:
Maybe add some description for this session, like "Choose the necessary framework dependencies to install based on your deploy environment. "